File: Object_Orianted_Programming_For_YTM/Java/ClassExample/src/SporAraba.java
public class SporAraba  extends Araba/*İnheritance */ {
    boolean turbo;
    
    public SporAraba(String marka, String model, boolean turbo) {
        super(marka, model);
        this.turbo = turbo;
    }
    public void hareketEt() {
        System.out.println("Spor araba hızlı bir şekilde hareket ediyor.");
    }
}

File: Object_Orianted_Programming_For_YTM/Python/ClassExample/src/SporAraba.py
import Araba
class SporAraba(Araba):
    def __init__(self, marka, model, turbo):
        super().__init__(marka, model)
        self.turbo = turbo

    def hareket_et(self):
        print("Spor araba hızlı bir şekilde hareket ediyor.")